### PR TITLE
Update Volumes, fix navigation

### PR DESCRIPTION
**Pull request by usha-mandya**
Signed-off-by: Usha Mandya <usha.mandya@docker.com>

- [x] Refresh the Volumes topic
- [x] Move topics related the daemon to the Configure daemon section
- [x] Move storage drivers to config daemon section

~~TBD:~~

~~- [] Revamp the Storage overview topic~~
~~- [] Review reqs for the Troubleshoot topic~~

I'll create a follow-up PR to address the TBD items

### CODE REVIEW

File: _data/toc.yaml [hunk 1]
```diff
@@ -329,50 +355,22 @@ guides:
...
-        title: Use the Btrfs storage driver
-      - path: /storage/storagedriver/device-mapper-driver/
-        title: Use the Device mapper storage driver
-      - path: /storage/storagedriver/overlayfs-driver/
-        title: Use the OverlayFS storage driver
-      - path: /storage/storagedriver/zfs-driver/
-        title: Use the ZFS storage driver
-      - path: /storage/storagedriver/vfs-driver/
-        title: Use the VFS storage driver
+      title: Troubleshoot
```
Comment (usha-mandya) [inline _data/toc.yaml:373]: @thaJeztah Notes from my meeting with you suggests that discussed to move the Troubleshoot section to Bind mounts. Looks like the troubleshoot topic contains info on both bind mounts and volumes. Wondering if the move still makes sense. Could you confirm? Thanks.

File: storage/volumes.md [hunk 2]
```diff
@@ -228,11 +228,10 @@ volumes:
...
-volume will be reused on following invocations.
+Running `docker compose up` for the first time creates a volume. The same volume is reused when you subsequently run the command.
 
-A volume may be created directly outside of compose with `docker volume create` and
-then referenced inside `docker-compose.yml` as follows:
+You can create a volume directly outside of Compose using `docker volume create` and
+then reference it inside `docker-compose.yml` as follows:
 
 ```yaml
 version: "{{ site.compose_file_v3 }}"
```
Comment (usha-mandya) [inline storage/volumes.md:237]: Guess this isn't valid as per the Compose spec?

File: storage/volumes.md [hunk 3]
```diff
@@ -276,32 +271,30 @@ ID                  NAME                IMAGE               NODE
...
-Remove the service, which stops all its tasks:
+You can remove the service to stops the running tasks:
 
 ```console
 $ docker service rm devtest-service
 ```
 
-Removing the service does not remove any volumes created by the service.
-Volume removal is a separate step.
+Removing the service doesn't remove any volumes created by the service. Volume removal is a separate step.
```
Comment (thaJeztah) [inline storage/volumes.md:280]: Looks like you lost wrapping here

File: storage/volumes.md [hunk 4]
```diff
@@ -276,32 +271,30 @@ ID                  NAME                IMAGE               NODE
...
+If you start a container which creates a new volume, and the container
+has files or directories in the directory to be mounted such as `/app/`,
 the directory's contents are copied into the volume. The container then
 mounts and uses the volume, and other containers which use the volume also
 have access to the pre-populated content.
 
-To illustrate this, this example starts an `nginx` container and populates the
-new volume `nginx-vol` with the contents of the container's
-`/usr/share/nginx/html` directory, which is where Nginx stores its default HTML
+To illustrate this, the following example starts an `nginx` container and populates the new volume `nginx-vol` with the contents of the container's
```
Comment (thaJeztah) [inline storage/volumes.md:296]: Same here (wrapping)

File: storage/volumes.md [hunk 5]
```diff
@@ -349,14 +342,13 @@ $ docker volume rm nginx-vol
 
 For some development applications, the container needs to write into the bind
 mount so that changes are propagated back to the Docker host. At other times,
-the container only needs read access to the data. Remember that multiple
-containers can mount the same volume, and it can be mounted read-write for some
-of them and read-only for others, at the same time.
+the container only needs read access to the data. Multiple
+containers can mount the same volume. You can simultaneously mount a single volume as `read-write` for some containers and as `read-only` for others.
```
Comment (thaJeztah) [inline storage/volumes.md:346]: And here

File: storage/volumes.md [hunk 6]
```diff
@@ -416,9 +408,9 @@ $ docker container rm nginxtest
 $ docker volume rm nginx-vol
 ```
 
-## Share data among machines
+## Share data with machines
```
Comment (thaJeztah) [inline storage/volumes.md:411]: I think the intent here was to describe how to share data on multiple machines; not sure if "with" covers that.

Perhaps "between" could be an alternative?

File: storage/volumes.md [hunk 7]
```diff
@@ -467,10 +459,12 @@ $ docker volume create --driver vieux/sshfs \
 
 ### Start a container which creates a volume using a volume driver
 
-This example specifies a SSH password, but if the two hosts have shared keys
-configured, you can omit the password. Each volume driver may have zero or more
-configurable options. **If the volume driver requires you to pass options, you
-must use the `--mount` flag to mount the volume, rather than `-v`.**
+The following example specifies an SSH password. However, if the two hosts have shared keys configured, you can exclude the password. Each volume driver may have zero or more configurable options.
```
Comment (thaJeztah) [inline storage/volumes.md:462]: Can you wrap this line as well?

File: storage/volumes.md [hunk 8]
```diff
@@ -537,41 +533,41 @@ Then in the next command, we:
...
 
 ## Remove volumes
 
-A Docker data volume persists after a container is deleted. There are two types
+A Docker data volume persists after you delete a container. There are two types
 of volumes to consider:
 
-- **Named volumes** have a specific source from outside the container, for example `awesome:/bar`.
-- **Anonymous volumes** have no specific source so when the container is deleted, instruct the Docker Engine daemon to remove them.
+- Named volumes have a specific source from outside the container, for example `awesome:/bar`.
```
Comment (thaJeztah) [inline storage/volumes.md:564]: Does this need a comma after "for example"?

```suggestion
- Named volumes have a specific source from outside the container, for example, `awesome:/bar`.
```

**Review by thaJeztah — COMMENTED** [review]
left some comments/suggestions, otherwise looks good, thanks!

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**